### PR TITLE
Random Change

### PR DESCRIPTION
**Pull request by 1798643961**
Random Change

### CODE REVIEW

File: patches/minecraft/net/minecraft/entity/Entity.java.patch [hunk 1]
```diff
@@ -169,20 +169,6 @@
...
-+         ((org.bukkit.craftbukkit.v1_16_R3.entity.CraftPlayer) this.bukkitEntity).setHandle((net.minecraft.entity.player.ServerPlayerEntity) this);
-+      } else if (bukkitEntity instanceof org.bukkit.craftbukkit.v1_16_R3.entity.CraftHumanEntity && this instanceof net.minecraft.entity.player.PlayerEntity) {
-+         ((org.bukkit.craftbukkit.v1_16_R3.entity.CraftHumanEntity) this.bukkitEntity).setHandle((net.minecraft.entity.player.PlayerEntity) this);
-+      } if (bukkitEntity instanceof org.bukkit.craftbukkit.v1_16_R3.entity.CraftLivingEntity && this instanceof net.minecraft.entity.LivingEntity) {
-+         ((org.bukkit.craftbukkit.v1_16_R3.entity.CraftLivingEntity) this.bukkitEntity).setHandle((net.minecraft.entity.LivingEntity) this);
-+      } else {
-+         this.bukkitEntity.setHandle(this);
-+      }
-+   }
-+
```
Comment (Romaindu35) [inline patches/minecraft/net/minecraft/entity/Entity.java.patch]: Why remove recent fix ?
You move this in other class ?

File: src/main/java/com/mohistmc/eventhandler/dispatcher/WorldEventDispatcher.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import org.bukkit.craftbukkit.v1_16_R3.CraftServer;
+import org.bukkit.craftbukkit.v1_16_R3.CraftWorld;
+import org.bukkit.event.world.WorldLoadEvent;
+import org.bukkit.event.world.WorldSaveEvent;
+
+public class WorldEventDispatcher {
+
+    //For WorldLoadEvent
+    @SubscribeEvent(receiveCanceled = true)
+    public void onWorldLoadEvent(WorldEvent.Load event) {
```
Comment (Romaindu35) [inline src/main/java/com/mohistmc/eventhandler/dispatcher/WorldEventDispatcher.java:14]: Why use forge event ? To make this compatible with forge mod ?

Comment (1798643961) [reply src/main/java/com/mohistmc/eventhandler/dispatcher/WorldEventDispatcher.java:14]: Yep

File: patches/minecraft/net/minecraft/entity/Entity.java.patch (hunk 1 above)
Comment (1798643961) [reply patches/minecraft/net/minecraft/entity/Entity.java.patch]: I have moved it，I found it is no need to add this method，just

Comment (1798643961) [reply patches/minecraft/net/minecraft/entity/Entity.java.patch]: Just move it to the Entity.class

Comment (1798643961) [reply patches/minecraft/net/minecraft/entity/Entity.java.patch]: Just move it to the Entity.class

Comment (1798643961) [reply patches/minecraft/net/minecraft/entity/Entity.java.patch]: `+               // CraftBukkit start - Forward the CraftEntity to the new entity
+               // Mohist Start
+               entity.bukkitEntity = this.getBukkitEntity();
+               if (bukkitEntity instanceof org.bukkit.craftbukkit.v1_16_R3.entity.CraftPlayer && this instanceof net.minecraft.entity.player.ServerPlayerEntity) {
+                  ((org.bukkit.craftbukkit.v1_16_R3.entity.CraftPlayer) this.bukkitEntity).setHandle((net.minecraft.entity.player.ServerPlayerEntity) this);
+               } else if (bukkitEntity instanceof org.bukkit.craftbukkit.v1_16_R3.entity.CraftHumanEntity && this instanceof net.minecraft.entity.player.PlayerEntity) {
+                  ((org.bukkit.craftbukkit.v1_16_R3.entity.CraftHumanEntity) this.bukkitEntity).setHandle((net.minecraft.entity.player.PlayerEntity) this);
+               } if (bukkitEntity instanceof org.bukkit.craftbukkit.v1_16_R3.entity.CraftLivingEntity && this instanceof net.minecraft.entity.LivingEntity) {
+                  ((org.bukkit.craftbukkit.v1_16_R3.entity.CraftLivingEntity) this.bukkitEntity).setHandle((net.minecraft.entity.LivingEntity) this);
+               } else {
+                  this.getBukkitEntity().setHandle(entity);
+               }
+               // Mohist end
+               if (this instanceof MobEntity) {
+                  ((MobEntity)this).func_110160_i(true, true); // Paper drop lead
+               }
+               // CraftBukkit end`

Comment (1798643961) [reply patches/minecraft/net/minecraft/entity/Entity.java.patch]: `               // CraftBukkit start - Forward the CraftEntity to the new entity
              // Mohist Start
            entity.bukkitEntity = this.getBukkitEntity();
            if (bukkitEntity instanceof org.bukkit.craftbukkit.v1_16_R3.entity.CraftPlayer && this instanceof net.minecraft.entity.player.ServerPlayerEntity) {
             ((org.bukkit.craftbukkit.v1_16_R3.entity.CraftPlayer) this.bukkitEntity).setHandle((net.minecraft.entity.player.ServerPlayerEntity) this);
           } else if (bukkitEntity instanceof org.bukkit.craftbukkit.v1_16_R3.entity.CraftHumanEntity && this instanceof net.minecraft.entity.player.PlayerEntity) {
                ((org.bukkit.craftbukkit.v1_16_R3.entity.CraftHumanEntity) this.bukkitEntity).setHandle((net.minecraft.entity.player.PlayerEntity) this);
              } if (bukkitEntity instanceof org.bukkit.craftbukkit.v1_16_R3.entity.CraftLivingEntity && this instanceof net.minecraft.entity.LivingEntity) {
                 ((org.bukkit.craftbukkit.v1_16_R3.entity.CraftLivingEntity) this.bukkitEntity).setHandle((net.minecraft.entity.LivingEntity) this);
              } else {
                 this.getBukkitEntity().setHandle(entity);
              }
              // Mohist end
             if (this instanceof MobEntity) {
                  ((MobEntity)this).func_110160_i(true, true); // Paper drop lead
               }
               // CraftBukkit end`
I changed it to this

**Review by Romaindu35 — APPROVED** [review]

**Pull request closed without merging**